src/csharp/BUILD-INTEGRATION.md

```diff
@@ -352,6 +360,7 @@ The following metadata are recognized on the `<Protobuf>` items.
 | GrpcOutputDir  | See notes | A directory                        | Directory for generated gRPC stubs    |
 | GrpcOutputOptions | | arbitrary options                  | Extra options passed to gRPC codegen as `--grpc_opt=opt1,opt2` |
 | GrpcServices   | `both`    | `none`, `client`, `server`, `both` | Generated gRPC stubs             |
+| AdditionalImportDirs | See notes | Directories                        | Additional directories for to find files |
```

Comment:
typo: "Additional directories for to find files"? Let's update to something more intelligible (also see the comment below).

Reply:
updated